### ISSUE TITLE
Required OpenAPI/Swagger field "version" missing

### ISSUE THREAD

**Issue by assert-not-singularity** [openhab/openhab-core#1826]
The [OpenAPI specification](https://swagger.io/specification/) requires the specification file to have a version number ([see here](https://swagger.io/specification/#info-object)). However, the generated json file does only have a title given:

```json
{
  "swagger": "2.0",
  "info": {
    "title": "openHAB REST API"
   },
}
```

When trying to parse the specification with NSwagStudio for example, one will receive an error. However, when adding the required field

```json
{
  "swagger": "2.0",
  "info": {
    "title": "openHAB REST API",
    "version": "2.5.9"
  },
}
```

the code will compile successfully.

**Comment by wborn** [comment]
Does it work with OH3?

It should have an API version, see:

```json
{
   "openapi": "3.0.1",
   "info": {
      "title": "openHAB REST API",
      "contact": {
         "name": "openHAB",
         "url": "https://www.openhab.org/docs/"
      },
      "version": "4"
   },

```

**Comment by assert-not-singularity** [comment]
> 
> 
> Does it work with OH3?
> 
> It should have an API version, see:
> 
> ```json
> {
>    "openapi": "3.0.1",
>    "info": {
>       "title": "openHAB REST API",
>       "contact": {
>          "name": "openHAB",
>          "url": "https://www.openhab.org/docs/"
>       },
>       "version": "4"
>    },
> ```

Could you give me the link to the OpenAPI definition, please? In contrast to openHAB 2, the path to the json file is not shown in the API browser.

**Comment by wborn** [comment]
It's available at: /rest/spec

**Comment by assert-not-singularity** [comment]
Yes, in openHAB 3, there is a version number. 🙂

**Comment by wborn** [comment]
Great! It's probably not easy to add in 2.5.x because the spec is generated by the core code which is frozen in 2.5.x.

**Comment by wborn** [comment]
I'll close this since the issue is fixed in 3.0 and there won't be any fixes in OH 2.5.x core.
